app/__init__.py

```diff
@@ -68,9 +72,9 @@ class Meta:
 
 @app.route('/api/timeline_post', methods=['POST'])
 def post_time_line_post():
-    name = request.form['name']
-    email = request.form['email']
-    content = request.form['content']
+    name = request.form.get('name', "")
```

Comment:
This change is to prevent NULL errors with the peewee functions in the tests. The `, ""` is the default when there is no 'name' field in the request.